blueprints/token.lua

```diff
@@ -57,6 +57,7 @@ Variant = "0.0.3"
 
 -- token should be idempotent and not change previous state updates
 Balances = Balances or { [ao.id] = utils.toBalanceValue(10000 * 1e12) }
+TotalSupply = TotalSupply or utils.toBalanceValue(10000 * 1e12)
```

Comment:
This should also probably use `10000 * 10 ^ Denomination` instead of `10000 * 1e12` for flexibility.